cpp/include/IceUtil/Config.h

```diff
@@ -109,7 +109,7 @@
 //
 #if (ICE_CPLUSPLUS >= 201103) || \
     ((defined(__GNUC__) && defined(__GXX_EXPERIMENTAL_CXX0X__) && ((__GNUC__* 100) + __GNUC_MINOR__) >= 405)) || \
```

Comment:
Not clear that we need to keep this branch with `__GXX_EXPERIMENTAL_CXX0X__`

Reply:
We'll want to clean this up anyway soon since all our compilers will be c++11 at least.

Reply:
I'll leave it like it is for now.